tgui/packages/tgui/interfaces/CommandTablet.js

```diff
@@ -64,10 +64,23 @@ export const CommandTablet = (_props, context) => {
                     evacuation procedures.
                   </NoticeBox>
                 )}
+                <Flex.Item>
+                  <Button.Confirm
+                    fluid={1}
+                    color="orange"
+                    icon="phone-volume"
+                    content={"Send Distress Beacon"}
+                    confirmColor="bad"
+                    confirmContent="Confirm?"
+                    confirmIcon="question"
+                    onClick={() => act('distress')}
+                    disabled={!canEvac} />
```

Comment:
I would advise using a different const as this makes it impossible to call distress while evac is ongoing